app/src/ui/lib/button.tsx

```diff
@@ -176,6 +179,7 @@ export class Button extends React.Component<IButtonProps, {}> {
         aria-disabled={disabled ? 'true' : undefined}
         aria-label={this.props.ariaLabel}
         aria-haspopup={this.props.ariaHaspopup}
+        autoFocus={this.props.autoFocus}
```

Comment:
Yes. I set it to warn... but to only warn when using on non react components (so it wasn't super noisy). I can completely turn it off.. but was a bit torn since there are valid cases for it so it something to be aware of, and  accessibility team is planning to have some guidance on this soon.